### ISSUE TITLE
3.0.0-beta.1 not found in CocoaPods

### ISSUE THREAD

**Issue by AleLudovici** [ReactiveX/RxSwift#880]
**Short description of the issue**:

  3.0.0-beta.1 not available found in CocoaPods

**Self contained code example that reproduces the issue**:

```swift
 # Podfile
use_frameworks!

target 'YOUR_TARGET_NAME' do
    pod 'RxSwift',    '~> 3.0.0-beta.1'
    pod 'RxCocoa',    '~> 3.0.0-beta.1'
end

# RxTests and RxBlocking make the most sense in the context of unit/integration tests
target 'YOUR_TESTING_TARGET' do
    pod 'RxBlocking', '~> 3.0.0-beta.1'
    pod 'RxTests',    '~> 3.0.0-beta.1'
end

  post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '3.0'
      config.build_settings['MACOSX_DEPLOYMENT_TARGET'] = '10.10'
    end
  end
end
```

**Xcode version**:

```
  Xcode 8 GM
```

**Expected outcome**:

  RxSwift should be installed

**What actually happens**:

CocoaPods output the following error: 
[!] Unable to satisfy the following requirements:

- `RxSwift (= 3.0.0-beta.1)` required by `Podfile` 

**Installation method**:
  - [x] CocoaPods
  - [ ] Carthage
  - [ ] Git submodules

**I have multiple versions of Xcode installed**:
  - [ ] yes (which ones)
  - [x] no

**Level of RxSwift knowledge**:
  - [ ] just starting
  - [ ] I have a small code base
  - [x] I have a significant code base

**Comment by dpassage** [comment]
Did you run `pod repo update`?

**Comment by riccardo-tesio** [comment]
I have the same issue.
Unfortunately `pod repo update` does not solve the problem.

**Comment by stack** [comment]
The release has not been tagged on GitHub, yet.

**Comment by kzaher** [comment]
Hi guys, 

I still haven't hit the release button for beta.1 (aka push the release tag). 

I wated to investigate some strange behavior I've seen. Wasn't sure is it an ios bug, our bug, or automation tools bug. 

I'll need a day or two to close that end since it's mid week.

**Comment by dpassage** [comment]
Actually, if you want to live dangerously, you can just grab the master branch with `pod 'RxSwift', :git => 'https://github.com/ReactiveX/RxSwift.git'`

But, again, that's living dangerously. :) That incantation will stick you onto the master every time you do `pod update`; you probably want to put in a `:tag => '3.0.0.beta.1` as soon as it drops.

**Comment by riccardo-tesio** [comment]
@dpassage thank you.

**Comment by AleLudovici** [comment]
@kzaher ok! thanks for the clarification

**Comment by kzaher** [comment]
Hi @AleLudovici ,

we've just published 3.0.0-beta.1 version that should work for you.

**Comment by awclives** [comment]
wonderful!
